getting-started/templates/systemlink-admin-values.yaml

```diff
@@ -2,6 +2,21 @@
 ## This is a YAML-formatted file.
 ## Declare variables to be passed into your templates.
 
+global:
```

Comment:
Why do we need this? The installation guide tells the user to provide all of systemlink-values, systemlink-admin-values, and systemlink-secrets when installing the Admin chart. We should not be duplicating values.

Reply:
Sadly is not possible to reuse the anchors from `systemlink-values.yaml` in any other chart. From the [Helm documentation](https://helm.sh/docs/chart_template_guide/yaml_techniques/#yaml-anchors) I noticed that it mentions that 

>While there are a few cases where anchors are useful, there is one aspect of them that can cause subtle bugs: The first time the YAML is consumed, the reference is expanded and then discarded.

Therefore, even if we specified all the value files, `strimzi-kafka-operator` does not get to see the value for `&imageRegistryRef` nor `&pullSecrets`, because the `systemlink-values.yaml` file was already parsed and the anchors were discarded. 
Maybe we can get around this issue by forking all the templates for the operator and reference the global values from there, like in [this line](https://github.com/strimzi/strimzi-kafka-operator/blob/main/helm-charts/helm3/strimzi-kafka-operator/templates/060-Deployment-strimzi-cluster-operator.yaml#L51).

Reply:
If we need to make this change then this is a breaking change and you need to update the installation guide to inform users that information must be duplicated between the two values files.

Reply:
Right, I followed the procedure described in the CONTRIBUTING.md and requested a change there. I didn't get any email or notification about it to follow up. I can sync up with @MelissaHilliard to ensure that the documentation is updated.